requirements-dev.txt

```diff
@@ -1,9 +1,9 @@
-black
-flake8
+black==23.1.0
```

Comment:
I pinned those dependencies also used by pre-commit so that they are always the same